Formula/qemu.rb

```diff
@@ -82,6 +75,8 @@ def install
       args << "--enable-cocoa"
     end
 
+    # fix xcode search version header error
+    mv "VERSION", "VERSION.txt"
```

Comment:
Is upstream aware of this bug?

Reply:
There is a patch in upstream pending review:

https://patchew.org/QEMU/CA+pCdY09+OQfXq3YmRNuQE59ACOq7Py2q4hqOwgq4PnepCXhTA@mail.gmail.com/

Reply:
Yes, related tickets:

- Gitlab: https://gitlab.com/qemu-project/qemu/-/issues/129
- Patchew: [https://patchew.org/QEMU/...](https://patchew.org/QEMU/CA+pCdY09+OQfXq3YmRNuQE59ACOq7Py2q4hqOwgq4PnepCXhTA@mail.gmail.com/)

Reply:
I just changed this PR to use qemu's patch.